Sketcher/SketchUnmap.FCMacro

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+#####################################
+# Copyright (c) openBrain 2019
+# Licensed under LGPL v2
+#
+# This macro will unmap a sketch from eg. a face and make its placement absolute in the body. It proposes 3 options (forcely the first if sketch not in a body) :
+# * "Raw" mode => the sketch placement is made absolute in the body referential, nothing more
+# * "DP@Face mode" => a datum plane is created where the mapping face is, then the sketch is attached to it respecting its attachment offset
+# * "DP@Sketch" mode => a datum place is created where the sketch is (including attachment offset), then the sketch is attached to its origin
+#
+# Main use of the macro is to make design more robust to topological naming issue
+#
+# Version history :
+# *0.5 : beta release
```

Comment:
please consider adding the latest revision from recent to oldest instead

Reply:
OK.